build.xml

```diff
@@ -129,7 +129,7 @@
     <property name="cassandra.test.processorCount" value="4"/>
 
     <!-- skip flushing schema tables during tests -->
-    <property name="cassandra.test.flush_local_schema_changes" value="true" />
+    <property name="cassandra.unsafesystem" value="true" />
```

Comment:
Could we make this name less generic? unsafe, but in what way?
Suppose I'd like to use this property in the code. When am I allowed to? When does it make sense?
I'd like this to be commented upon and more self-evident from the property name.
Do you think this can be done?

Reply:
I'd not change it mainly because it is named the same in the OSS (for some long time already)